### PR TITLE
Arpit/addteam

### PR DESCRIPTION
**Pull request by arpitt2102**
Added, moved, and removed team members

### CODE REVIEW

File: src/components/VolunteersForm.js [hunk 1]
```diff
@@ -163,11 +163,11 @@ export default class VolunteersForm extends React.Component {
                         {/* // )} */}
 
                           <div className="form-group">
-                              <label id="Heard_about_us" htmlFor="herd">How did you hear about Software For Love? (Select all that Apply) {required_star}</label>
+                              <label id="Heard" htmlFor="heard">How did you hear about Software For Love? (Select all that Apply) {required_star}</label>
```
Comment (Suri111200) [inline src/components/VolunteersForm.js:166]: Did the changes here work? If not, I would appreciate if you could push a commit reverting the changes as they were already on master

**Review by Suri111200 — CHANGES_REQUESTED** [review]
See comments. Please message on Discord if you have any further questions.

**Comment by Suri111200** [comment]
Also for future reference, link the associated issue to your pull request. It can be done here:
image

File: src/components/VolunteersForm.js (hunk 1 above)
Comment (arpitt2102) [reply src/components/VolunteersForm.js]: Done!

**Review by Suri111200 — APPROVED** [review]

**Pull request merged**